pytermgui/ansi_interface.py

```diff
@@ -12,17 +12,25 @@
 import re
 import sys
 import signal
+import asyncio
 
 from typing import Optional, Any, Union, Callable, Tuple
 from dataclasses import dataclass, fields
 from enum import Enum, auto as _auto
 from sys import stdout as _stdout
 from string import hexdigits
 from subprocess import run as _run, Popen as _Popen
-from os import name as _name, get_terminal_size, system
+from os import name as _name, system
+from shutil import get_terminal_size
+
+# This wraps `os.get_terminal_size` and adds few functions to
+#  try and get size if `os.get_terminal_size` fails
+#  and if all functions fail it fallbacks to (80, 24)
```

Comment:
Good comment for the reviewer (moi), but we should remove this before merging. Nice replacement though!

Reply:
I will remove it noe